### ISSUE TITLE
Plover 4 not recognising keystrokes from Planck Rev 6

### ISSUE THREAD

**Issue by bensleveritt** [openstenoproject/plover#1105]
## Summary

Plover 4 doesn't recognise key presses, when Plover 3 does.

## Reproducing

I have a Planck Rev 6 in steno mode, and Plover 3 connects and recognises keystrokes. In the same setup, Plover 4 doesn't seem to register key presses.

As QMK (Planck firmware) supports both Gemini and TX bolt protocols, I tried switching between them. Plover reports connected for both, which feels incorrect (surely it's one or the other).

## Plover Version

Plover 4.0.0.dev8+66.g685bd33 

## System

macOS Catalina 10.15.3

**Comment by JRJurman** [comment]
@bensleveritt what does your keymap file look like? I have a Planck EZ, and I'm not too terribly familiar with the different versions, but having that might help us understand what is different / not working.

**Comment by bensleveritt** [comment]
Here are my keymap files. I added a default protocol, but still no traction. https://github.com/bensleveritt/qmk_firmware/tree/tweak-default/keyboards/planck/keymaps/bensleveritt

**Comment by bensleveritt** [comment]
Hah. I think I've got it. I think the default Planck keymap doesn't output Steno keys, despite being in Plover mode, and perhaps somehow I got confused (possibly Plover was in QWERTY mode or something).

At any rate, taking the keys from the default Planck/Steno keymap, and merging it in seems to have done the trick.

https://github.com/qmk/qmk_firmware/commit/d0931fea3597dad9b5c6b45500a2aeded029ce86
